src/common/gui/RuntimeFont.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "vstgui/vstgui.h"
+#include "vstgui/lib/cfont.h"
+#include "vstgui/lib/platform/iplatformfont.h"
+
+namespace Surge
+{
+namespace GUI
+{
+
+/**
+ * initializeRuntimeFontFromDistribution
+ *
+ * The role of this function is to load a font from the bundle, dll, so whatever
+ * and plumb through the VSTGUI to create the 9 and 14 point fonts onto the two
+ * global variables "surge_minifont" and "surge_patchfont" if they are NULL
+ * at calltime.
+ *
+ * This is a remarkably OS-specific thing to do, so it is implemented in
+ * mac/win/linux RunTimeFontOS.cpp. (As of this writing it is neither implemented
+ * nor referenced on linux)
```

Comment:
Just one sentence would be enough: "The implementation is OS specific.". That's it. We don't want over explain this in every function will be like this. And we do not want to keep backlog of the implementation status here.